source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -688,5 +688,37 @@ than an element in the range being searched.
 
 The elements e of [start, end) must be partitioned with respect to the comparator used.
 
+.. code:: cpp
+
+  template <typename Policy, typename InputIt, typename OutputIt, typename UnaryOp,
+      typename UnaryPredicate>
+  OutputIt
+  transform_if(Policy&& policy, InputIt start, InputIt end, OutputIt result, UnaryOp op,
+      UnaryPredicate pred);                                                               // (1)
+
+  template <typename Policy, typename InputIt1, typename InputIt2, typename OutputIt, 
+      typename BinaryOp, typename BinaryPredicate>
+  OutputIt
+  transform_if(Policy&& policy, InputIt1 start1, InputIt1 end1, InputIt2 start2, OutputIt result,
+      BinaryOp op, BinaryPredicate pred);                                                 // (2)
+
+:code:`oneapi::dpl::transform_if` applies a given function to the elements of the input sequence(s) that
```

Comment:
Why are we using :code:'<value>' in this file and "<value>" in other files? I think that we should standardize this and am partial to "<value>".

Please add a colon after "...the algorithm"

Reply:
for :code:` ` vs  ` `, I don't have a good explanation other than staying consistent with the other cases in this file.  If we want to make a formatting change on this topic, I'd suggest we do it for the whole oneDPL spec, and in a separate PR from this.

Reply:
After a quick look around, I think that this file is the only one using :code: (for docs anyway). I would prefer that we stay consistent. I think that a new PR for this issue would be good to fix before the next release.

Reply:
To be clear, this is the oneDPL specification within the oneAPI specification in UXL, rather than docs which would be tied to an intel release.  
I see a few other instances of `:code:` in `sycl_kernels_api.rst` and `common.rst`, but most are here in this file.  However, I agree it would be good to update with a PR prior to the specification freeze.

Reply:
That is good to keep in mind!

Would you like me to create a PR? I can have that done by EOD.

Reply:
Sure, if you are willing.  Unless there is objection from others for some reasons I don't know about it should be easy to approve.

Reply:
I'll get it started, and if there are objections, we can always delete/modify the PR.

Reply:
At the HTML level, this is formatting for backquotes:
```
<code class="docutils literal notranslate">
<span class="pre">using</span> <span class="pre">scalar_type</span> <span class="pre">=</span> <span class="pre">result_type;</span>
</code>
```
and this is the formatting for `::code:`
```
<code class="code docutils literal notranslate">
<span class="pre">namespace</span> <span class="pre">oneapi::dpl::execution</span>
</code>
```
(the examples use different strings - I have not found the same or similar strings for both styles.)

The difference seems to be only in the extra "code" class name in attributes.. I guess it technically allows to set different formatting in CSS or JavaScript, but I do not know if it is used in the oneAPI specification. Visually both styles look the same.

Reply:
I looked through the other products in this repo and we are the only ones using the :code:'value' (single quote)and only in some spots. Most of the instances are "value"(no :code: and with double quotes). We should stick to one. 

I have created a PR, but do not have approval/access to push it. Can someone on this thread give me write access for this repo?

Reply:
As far as I can tell, contributions/PRs to this repo are done from forks, not from repo branches.
If you do not want to fork it in your own GitHub account, I can create a PR from my fork.

Reply:
I have followed up via email. Thank you @akukanov

Reply:
https://github.com/uxlfoundation/oneAPI-spec/pull/565